tests/integration/security/multi_signers/main_test.go

```diff
@@ -0,0 +1,121 @@
+// +build integ
```

Comment:
One request here: could we just overwrite the existing security/external_ca tests with this new code. The functionality tested is almost exactly the same, and this is the latest version of that functionality

Reply:
done~